### ISSUE TITLE
Question about pretrain.py

### ISSUE THREAD

**Issue by hhyhcr449** [jimmykobe/CVPR2019-MADDoG#4]
Thanks for your open source code.

When I read your implement of the function **Pre_train**,I found that you use `torch.nn.MSELoss`

It is not correspond to the implement of the paper，which uses K-way classification with a crossentropy loss and does not use `DepthEsmator`.
It confused me for a lot time.

Thank you for your help.

**Comment by rshaojimmy** [comment]
Hi,

Yes, you can also use the K-way classification with a cross-entropy loss which described in our paper. Two ways of pre-training have similar results.

It depends on the definition of discriminative feature space, which can pre-trained by the supervision of labels (cross-entropy loss) or by the depth maps (mse loss). Two ways of pre-training can all contributes to the discriminative feature spaces.

Thanks.
